cli/src/pcluster/config/cluster_config.py

```diff
@@ -1114,12 +1119,23 @@ def _register_validators(self, context: ValidatorContext = None):  # noqa: D102
             self._register_validator(IamResourcePrefixValidator, resource_prefix=self.resource_prefix)
 
 
+class OneApi(Resource):
+    """Represent the OneApi configuration."""
+
+    def __init__(self, base_toolkit: bool = None, hpc_toolkit: bool = None):
+        super().__init__()
+        self.base_toolkit = Resource.init_param(base_toolkit or hpc_toolkit, default=False)
+        self.hpc_toolkit = Resource.init_param(hpc_toolkit, default=False)
+
+
 class IntelSoftware(Resource):
-    """Represent the Intel select solution configuration."""
+    """Represent the Intel software configuration."""
 
-    def __init__(self, intel_hpc_platform: bool = None):
+    def __init__(self, one_api: OneApi = None, intel_hpc_platform: bool = None, python: bool = None):
         super().__init__()
         self.intel_hpc_platform = Resource.init_param(intel_hpc_platform, default=False)
+        self.one_api = Resource.init_param(one_api)
+        self.python = python
```

Comment:
Did PM approve this to be `Python` rather than `IntelPython`?
I'm asking because for HpcPlatform we preferred to specify Intel as prefix even if it is nested in IntelSoftwareSection.

Reply:
Yes. PM has reviewed the design doc